appveyor.yml

```diff
@@ -0,0 +1,38 @@
+environment:
+  matrix:
+    - nodejs_version: 6
+      test_suite: "simple"
+    - nodejs_version: 6
+      test_suite: "installs"
+    - nodejs_version: 6
+      test_suite: "kitchensink"
+    - nodejs_version: 4
+      test_suite: "simple"
+    - nodejs_version: 4
+      test_suite: "installs"
+    - nodejs_version: 4
+      test_suite: "kitchensink"
+    - nodejs_version: 0.10
+      test_suite: "simple"
+
+cache:
+  - node_modules
+  - packages\react-scripts\node_modules
+
+clone_depth: 50
+
+matrix:
+  fast_finish: true
```

Comment:
Since AppVeyor limits to 1 concurrent job (not 5 like travis) it's probably in our best interest to kill the entire matrix after the first failure instead of wasting time testing the rest.